gallery/src/styles/NxCard/NxCardHorizontalExample.tsx

```diff
@@ -0,0 +1,102 @@
+/*
+ * Copyright (c) 2019-present Sonatype, Inc.
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
+ */
+import React from 'react';
+import { faShapes, faEdit, faTrash } from '@fortawesome/free-solid-svg-icons';
+
+import { NxFontAwesomeIcon, NxButton } from '@sonatype/react-shared-components';
+
+export default function NxCardHorizontalExample() {
+  return (
+    <div className="nx-card-container nx-card-container--column">
+      <section className="nx-card">
+        <header className="nx-card__header">
+          <h3 className="nx-h3">Header</h3>
+        </header>
+        <div className="nx-card__call-out">
+          <span className="nx-card__call-out-text">42</span>
+        </div>
+        <div className="nx-card__content">
+          Text
+        </div>
+        <footer className="nx-card__footer">
+          <a href="#" className="nx-text-link">Link</a>
+        </footer>
+      </section>
+      <section className="nx-card">
+        <header className="nx-card__header">
+          <h3 className="nx-h3">Card header</h3>
+        </header>
+        <div className="nx-card__call-out">
+          <NxFontAwesomeIcon icon={faShapes} />
+        </div>
+        <div className="nx-card__content">
+          Data point details
+        </div>
+        <footer className="nx-card__footer">
+          <div className="nx-btn-bar">
+            <NxButton variant="icon-only"><NxFontAwesomeIcon icon={faEdit} className="nx-icon--large" /></NxButton>
```

Comment:
`.nx-icon--large` doesn't do anything here (and in other buttons below)

Reply:
> `.nx-icon--large` doesn't do anything here (and in other buttons below)

Fixed here: https://github.com/sonatype/sonatype-react-shared-components/pull/170/commits/cd45aea566dbdab783874f540aa041f78f82fdf2